BUILD.gn

```diff
@@ -99,6 +97,52 @@ if (!is_android) {
   } else {
     library_type = "static_library"
   }
+  support_unknown_function_handling = false
+  if (ar_path != "" && !is_win && (current_cpu == "arm64" || current_cpu == "x86_64")) {
```

Comment:
Should be: if (defined(ar_path) ...
Otherwise GN gen fails:
https://dawn-review.googlesource.com/c/dawn/+/202314
https://ci.chromium.org/ui/p/dawn/builders/try/mac-rel/b8739735338416185889/overview
https://logs.chromium.org/logs/dawn/buildbucket/cr-buildbucket/8739735338416185889/+/u/gn_gen/stdout
ERROR at //third_party/vulkan-loader/src/BUILD.gn:101:7: Undefined identifier
  if (ar_path != "" && !is_win && (current_cpu == "arm64" || current_cpu == "x86_64")) {
@zoddicus

Reply:
I have merged a PR with the `defined(ar_path)` fix.
Let me know if that fixes it or what further changes are required.